src/contributing/usage-statistics.md

```diff
@@ -0,0 +1,28 @@
+# Usage Statistics
+
+If you would like to contribute usage reports, the
+[PopCorn](https://github.com/the-maldridge/popcorn) program reports installation
+statistics back to the Void project. These statistics are purely opt-in, the
+reporting programs are *not* installed by default on any void systems.
+
+*PopCorn* only reports which packages are installed, their version, and the host
+CPU architecture (the output of `xuname`.) This does not report which services
+are enabled, or any other personal information. Individual systems are tracked
+persistently by a random (client generated) UUID.
```

Comment:
Please state that the purpose of this UUID is exclusively to permit accurate aggregation of statistics by ensuring each system is counted exactly once per sampling interval, which is 24 hours.